api/Entities/ImageEntity.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace iTechArt.Hotels.Api.Entities
+{
+    public class ImageEntity
+    {
+        [Key]
+        public int Id { get; set; }
+        [Required]
+        [MaxLength(2000)]
+        public string Path { get; set; }
+        public bool IsOuterLink { get; set; }
+        public int HotelId { get; set; }
```

Comment:
Is it foreign key?

Reply:
Yes it is. There are three ways to show that something is a foreign key. The first one is to do it with attributes, the second one is to use fluent api and the last one is to use the way I use. It is described here https://docs.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key 
We have also discussed this point with Andrew